packages/sanity/src/core/form/studio/assetSource/DefaultSource.tsx

```diff
@@ -70,16 +123,18 @@ const DefaultAssetSource = function DefaultAssetSource(
 
       setIsLoading(true)
 
-      fetch$.current = versionedClient.observable
-        .fetch(buildQuery(start, end, assetTypeParam), {}, {tag})
-        .subscribe((result) => {
-          setIsLastPage(result.length < PER_PAGE)
-          // eslint-disable-next-line max-nested-callbacks
-          setAssets((prevState) => prevState.concat(result))
-          setIsLoading(false)
-        })
+      if (typeof accept !== 'undefined') {
```

Comment:
I'm struggling to understand how this fits in. Is it not valid to pass `accept: undefined`? If `accept` _is_ `undefined`, should we not still fetch assets? If it (for some reason) goes from `!undefined` to `undefined`, will it not then keep `isLoading` forever? I might be missing something obvious here.

Reply:
Part of an old solution that needed to be removed :) 
Sorry about that, removed ✅